src/sentry/api/endpoints/project_app_store_connect_credentials.py

```diff
@@ -366,13 +366,19 @@ class AppStoreConnectCredentialsValidateEndpoint(ProjectEndpoint):  # type: igno
         "appstoreCredentialsValid": true,
         "itunesSessionValid": true,
         "pendingDownloads": 123,
-        "itunesSessionRefreshAt": "YYYY-MM-DDTHH:MM:SS.SSSSSSZ" | null
+        "itunesSessionRefreshAt": "YYYY-MM-DDTHH:MM:SS.SSSSSSZ" | null,
+        "lastCheckedBuilds": "YYYY-MM-DDTHH:MM:SS.SSSSSSZ" | null
     }
     ```
 
     Here the ``itunesSessionRefreshAt`` is when we recommend to refresh the
     iTunes session, and ``pendingDownloads`` is the number of pending downloads,
     and an indicator if we do need the session to fetch new builds.
+
+    ``lastCheckedBuilds`` is when sentry last checked for new builds, regardless
+    of whether there were any or no builds in App Store Connect at the time, or
+    if the check failed due to an expired iTunes session. In other words, this
+    date may represent a failed check.
```

Comment:
might be good to clear up if we want failed checks to be included in this or not.